### ISSUE TITLE
[Quest] Saving Princess Stillpine

### ISSUE THREAD

**Issue by phikaro** [HellscreamWoW/Tracker#2461]
Issue is that I cannot make the High Chief Bristlelimb to appear so I can kill him and get the key and save the princess from her prison. And High Chief Bristlelimb is supposed to appear after killing 7-8 of the  mobs in the camp according to wowhead comments.

http://www.wowhead.com/quest=9667/saving-princess-stillpine

**Comment by Malae1** [comment]
Confirmed, it does appear that it is related to killing a certain amount before this Creature is spawned.

**Comment by phikaro** [comment]
Hmm well I killed more than 10 yesterday and he did not appear, perhaps I should try again. :dancers:

**Comment by phikaro** [comment]
Nope, I tried one more time and nothing, I killed even more mobs than before, it still not spawning.

**Comment by Malae1** [comment]
Merely was confirming the issue of him not spawning, sorry for the confusion.

**Comment by zwei7** [comment]
I killed 40 around in those 2 camps away from princess stillpine.

**Comment by SunRaysFalls** [comment]
having same freaking issue with this quest. Killed like 100 mobs where it say and not spawned at all.
